test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -125,16 +125,16 @@ Feature: Bind a single application to multiple services
                 -   group: stable.example.com
                     version: v1
                     kind: Backend
-                    name: internal-db-1sbr
+                    name: $scenario_id-internal
                     id: db1
                 -   group: stable.example.com
                     version: v1
                     kind: Backend
-                    name: external-db-1sbr
+                    name: $scenario_id-external
                     id: db2
             """
-        Then Service Binding "binding-request-1sbr" is ready
+        Then Service Binding "$scenario_id-binding" is ready
```

Comment:
since only single binding is created in this scenario, then you can avoid specifying the binding name:

```
Then Service Binding is ready

```

Reply:
Fixed in latest.